tests/SiteTest.php

```diff
@@ -19,7 +19,7 @@ protected function setUp()
 
         $keyInfo = static::$_client->server()->getKeyInfo();
 
-        if ((int)$keyInfo['lim_dom'] < 2) {
+        if ((int)$keyInfo['lim_dom'] > -1 && (int)$keyInfo['lim_dom'] < 2) {
```

Comment:
Please, create method/trait for this check (code duplication)

Reply:
Moved out into the Utility to act as a method of the separate class